Firmware/mmu2.cpp

```diff
@@ -511,6 +512,9 @@ void MMU2::manage_response(const bool move_axes, const bool turn_off_nozzle) {
         // - still running -> wait normally in idle()
         // - failed -> then do the safety moves on the printer like before
         // - finished ok -> proceed with reading other commands
+#ifdef WATCHDOG
+        wdt_reset();
```

Comment:
Manually setting the watchdog is never the solution. It means that the heaters are also not updated. The `manage_inactivity(true)` lower is also suspicious since that one calls the `MMU2::mmu2.mmu_loop();` internally

Reply:
Indeed I noticed it is that way also on current MK3 branch so I didn’t touch it.

Reply:
Perhaps we should be calling `manage_heater()` just before `manage_inactivity()`. It is done in most places in the firmware and it kick the watchdog for us. If I search for `manage_inactivity()` then they are usually used together like this:

```c
manage_heater();
manage_inactivity();
```

Reply:
I am getting random axis crashes after this change. This looks strange:
![image](https://user-images.githubusercontent.com/8218499/164912907-8904f5a0-0ba8-4af5-8327-fa59764529eb.png)

Reply:
I didn't see the axis problem again. The E32903 seems to be an error code, I fixed it by pushing the middle button (I think?) on the MMU and that made the unit re-home.